packages/plugins/minos-discovery-kong/tests/test_kong/test_middleware.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+from collections import (
+    namedtuple,
+)
+from uuid import (
+    uuid4,
+)
+
+from minos.networks import (
+    Request,
+    Response,
+)
+from minos.plugins.kong.middleware import (
+    middleware,
+)
+from tests.utils import (
+    InMemoryHttpRequest,
+)
+
+PROTOCOL = "http"
+
+
+async def _fn(request: Request) -> Response:
+    return Response(f"{await request.content()}bar")
+
+
+Raw = namedtuple("Raw", ["headers", "content"])
+
+
+class TestKongDiscoveryClient(unittest.IsolatedAsyncioTestCase):
```

Comment:
Could you add a test that checks the case in which `"X-Consumer-Custom-ID"` is not provided?

Reply:
Done!